port/main.c

```diff
@@ -121,10 +133,69 @@ static int writeStatus(char *buffer, size_t bufSize, size_t offset)
 			SNPRINTF_APPEND("%2s%d_media=%s\n", netif->name, netif->num, drv->media(netif));
 	}
 
+#if LWIP_DNS
+	for (i = 0; i < LWIP_DHCP_MAX_DNS_SERVERS; ++i)
+		SNPRINTF_APPEND("dns_%u=%s\n", i, inet_ntoa(*dns_getserver(i)));
```

Comment:
not connected to this PR: what will happen if we would setup 2 ethernet ports to DHCP? I think everything will work apart from (potentially) DNS servers?

Reply:
DNS servers are overwritten by DHCP .

Reply:
thx for the info